src/main/java/seedu/address/commons/events/ui/PersonPanelDeselectionEvent.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+
+/**
+ * Represents a deselection in the Person List Panel
+ */
+public class PersonPanelDeselectionEvent extends BaseEvent {
+    @Override
+    public String toString() {
+        return getClass().getSimpleName();
+    }
+
```

Comment:
nit: extra empty line

Reply:
Resolved in b365ff455ce3f546938afb59d080631c1074b27b